### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Phil9904** [JustCarmen/webtrees-simple-media-display#5]
I am using Webtrees 2.1.17 and have the latest version of "webtrees-simple-media-display" installed.
With all the media objects that have more than one image, only the first image shows up under the "Facts and Events" tab.
I also have the "Vesta Facts and events" add-on module installed, and that shows the same results.

**Comment by JustCarmen** [comment]
Hi, I cannot reproduce this. In my case the module does work, with or without "Vesta Facts and Events".

I need more information to find the cause. Is it possible to see a page on your website where this is not working?

**Comment by Phil9904** [comment]
After further research, I have discovered that the add-on module "RepositoryHierachy" by Jefferson49 recently released an update, and that has caused the issue. When I deactivate that module, your Simple-media-display works fine.
  I have not notified Jefferson49 of this.
Thanks for responding and looking into it.
Your modules are great, and I use all of them!!
Phil

**Comment by JustCarmen** [comment]
I found out that one option in the RepositoryHierarchy module conflicts with this module:

> Directly show media objects of sources in facts and not subordinated within source citations.

Disable this option to use both modules.

Maybe @Jefferson49 can update his module to show multiple images from one media object in case this option is enabled.

I'll close this issue since it not something that should be solved within this module.

**Comment by Jefferson49** [comment]
In order to make both of the module features available in parallel, I integrated the code from [webtrees-simple-media-display](https://github.com/JustCarmen/webtrees-simple-media-display) into [RepositoryHierarchy](https://github.com/Jefferson49/RepositoryHierarchy).

@Phil9904
I provided a beta version at: https://github.com/Jefferson49/RepositoryHierarchy/issues/38 .
If you have some time, it would be helpful, if you could test the beta version an provide some feedback if it is working.
Probably, webtrees-simple-media-display should not be activated in this case.

**Comment by Jefferson49** [comment]
@JustCarmen 
I integrated the code from [webtrees-simple-media-display](https://github.com/JustCarmen/webtrees-simple-media-display) into [RepositoryHierarchy](https://github.com/Jefferson49/RepositoryHierarchy) and are about to release the new module version.

During final review, I recognized a file [**script.phtml**](https://github.com/JustCarmen/webtrees-simple-media-display/blob/main/resources/views/script.phtml). I do not quite understand what the script is intended for. A comment says that it is related to the media list page. However, I could not see any effect of it.

Is the script in script.phtml needed at all? Or can I simply delete it?

**Comment by Jefferson49** [comment]
I published a new [release v1.3.5](https://github.com/Jefferson49/RepositoryHierarchy/releases/tag/v1.3.5) of Repository Hierarchy, which now includes the feature to show multiple images of media objects.

**Comment by JustCarmen** [comment]
> If you have some time, it would be helpful if you could test the beta version and provide feedback if it works.

> I have published a new [release v1.3.5](https://github.com/Jefferson49/RepositoryHierarchy/releases/tag/v1.3.5) of Repository Hierarchy, which now includes the feature to show multiple images of media objects.

Sorry for my late response. I have now downloaded the latest version and as far as I can see the changes regarding the media display are working well.

> Is the script in script.phtml needed at all? Or can I simply delete it?

I think it is no longer needed. Probably at the time of writing this plugin, webtrees was still showing all media files of one object on the media list page. This script was written to prevent that. I saw that you didn't copy that script. That seems fine to me. I will also remove it from this module.

**Comment by Jefferson49** [comment]
> I think it is no longer needed. Probably at the time of writing this plugin, webtrees was still showing all media files of one object on the media list page.

Thanx for the confirmation that the script is not needed. Like you mentioned, I did not include it. Should be o.k. now.
